### PR TITLE
Further improvements to Dynamics

### PR DESCRIPTION
**Pull request by Variable-ind**
1. Image brush obeys dynamics
2. changed uncheck.png
3. the UI for max-min of size is changed to starting and ending
4. A progress bar is added to thresholds that show pressure and velocity to get a better idea in setting zones

https://user-images.githubusercontent.com/77773850/220355068-71b73143-625b-4868-94a8-d167bf1d546c.mp4

### CODE REVIEW

**Comment by Variable-ind** [comment]
i have also added a changed signal in the global tool options that will reset the dynamics whenever a change is made

https://user-images.githubusercontent.com/77773850/220361700-415c20ca-51f3-4cee-ac2a-0a1b53249ff3.mp4

**Comment by Variable-ind** [comment]
Added progress bar to Thresholds
 
https://user-images.githubusercontent.com/77773850/220377377-48a4b8b0-82c1-4424-9ea8-c718a6b1d2c2.mp4

**Review by OverloadedOrama — CHANGES_REQUESTED** [review]
Besides the comments I've left, everything looks pretty good!

File: src/UI/GlobalToolOptions/GlobalToolOptions.gd [hunk 1]
```diff
@@ -53,6 +57,19 @@ func _ready() -> void:
...
 
 
+func _input(event: InputEvent) -> void:
+	var pressure_rect := pressure_preview.get_global_rect()
+	var velocity_rect := velocity_preview.get_global_rect()
+	pressure_preview.value = 0
+	velocity_preview.value = 0
+	if event is InputEventMouseMotion:
+		if Input.is_action_pressed("left_mouse"):
+			if pressure_rect.has_point(get_global_mouse_position()):
```
[the remark this answers is not in the document]
Comment (Variable-ind) [reply src/UI/GlobalToolOptions/GlobalToolOptions.gd]: Done

File: src/UI/GlobalToolOptions/GlobalToolOptions.gd [hunk 2]
```diff
@@ -1,5 +1,7 @@
 extends PanelContainer
 
+signal changed
```
[the remark this answers is not in the document]
Comment (Variable-ind) [reply src/UI/GlobalToolOptions/GlobalToolOptions.gd]: Done

**Review by OverloadedOrama — APPROVED** [review]
Looks good, thank you!

**Pull request merged**